save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -141,7 +142,10 @@ class FixPlugin(
             }
 
     private fun createTestFile(path: Path, generalConfig: GeneralConfig): Path {
-        val pathCopy: Path = constructPathForCopyOfTestFile(FixPlugin::class.simpleName!!, path)
+        val pathCopy: Path = constructPathForCopyOfTestFile(
+            "${FixPlugin::class.simpleName!!}-${Random.nextInt()}",
```

Comment:
Now we can't retrieve this random value in `cleanupTempDir()`, may be at least add some fixme?

Reply:
Ok, I'll also link #156 to these `fixme`s

Reply:
I added (temporarily) this directory as a mutable class property. This way it can be properly cleaned up